tests/test_zootasks.py

```diff
@@ -4,27 +4,61 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+"""
+Test common developer mistakes in the model zoo and task list.
+
+Mostly just ensures the docs will output nicely.
+"""
+
 import os
 import unittest
 import parlai.core.testing_utils as testing_utils
+from parlai.zoo.model_list import model_list
+from parlai.tasks.task_list import task_list
+
+ZOO_EXCEPTIONS = {"fasttext_cc_vectors", "fasttext_vectors", "glove_vectors", "bert"}
 
 
 class TestZooAndTasks(unittest.TestCase):
     """Make sure the package is alive."""
 
-    def test_zoolist_types(self):
-        from parlai.zoo.model_list import model_list
+    def _assertZooString(self, member, container, animal_name=None):
+        msg = f'Missing or empty {member} in parlai.zoo.model_list'
+        if animal_name:
+            msg += f'[{animal_name}]'
```

Comment:
super nit: perhaps include a space here to format error msg better (i.e. `... in parlai.zoo.model_list [{...`)